CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Change Log
 
+## 1.15.2 - 2024-08-08
+
+### Added
```

Comment:
I'd argue this is engineering too since it doesn't impact end-users for this extension. But up to you

Reply:
Yeah I went back and forth on it. If anyone is a consumer of the API though, it may be considered a "feature add". I'm pretty indifferent though.